src/firmware_variables/variables.py

```diff
@@ -78,11 +78,11 @@ def set_variable(name, value, namespace=GLOBAL_NAMESPACE, attributes=DEFAULT_ATT
         raise WinError()
 
 
-def delete_variable(name, *args):
+def delete_variable(name, **kwargs):
```

Comment:
We need to allow positional too:
```suggestion
def delete_variable(name, *args, **kwargs):
```